src/app/application.cpp

```diff
@@ -146,6 +147,7 @@ Application::Application(const QString &id, int &argc, char **argv)
 #if !defined(DISABLE_GUI)
     setAttribute(Qt::AA_UseHighDpiPixmaps, true);  // opt-in to the high DPI pixmap support
     setQuitOnLastWindowClosed(false);
+    setDesktopFileName("org.qbittorrent.qBittorrent");
```

Comment:
Just want to make sure, you have tested it with wayland and this line is required, right?

Reply:
It is: https://github.com/flatpak/qt-flatpak-demo/commit/d2acabc5357767577000c6b781b39d43ede1ecda

Reply:
OK, then there is another thing, add the `#if` conditional as in https://github.com/notepadqq/notepadqq/commit/1fe0cc575aea8c6f67a649fc57b7f8adaf2a9805 as qbt still support Qt 5.5.1